tests/MSBuildDeviceIntegration/Tests/InstallAndRunTests.cs

```diff
@@ -1075,5 +1075,24 @@ public void SupportDesugaringStaticInterfaceMethods ()
 			);
 		}
 
+		[Test]
+		public void EnableAndroidStripILAfterAOT ()
```

Comment:
Possibly silly question, but is there any way to test that `$(AndroidStripILAfterAOT)`=True actually *does* something?  It could be a no-op and this test would still succeed!

Perhaps we should crack open the linked "app.dll" assembly and verify that e.g. `MainActivity.OnCreate()` has no body?

Reply:
Yes, we should do something like this test:
https://github.com/xamarin/xamarin-android/blob/5f38d099fc39fd270dca7f0d5ab57ff163945988/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Tasks/LinkerTests.cs#L293-L304

And then we could assert a method has 0 instructions?

Reply:
or a `null` method body, yes.

Reply:
That would be great, if we could create a test like that.

Reply:
@jonathanpeppers: `EnableAndroidStripILAfterAOT()` should also test the "profiled AOT/don't AOT everything" case.